core/mo-services-impl/ccsds-com-impl/src/main/java/esa/mo/com/impl/archive/db/PreparedStatements.java

```diff
@@ -51,8 +55,8 @@ public PreparedStatement getSelectAllCOMObjects() {
         return this.selectAllCOMObjectIds;
     }
 
-    public PreparedStatement getSelectCOMObjects() {
-        return this.selectCOMObjects;
+    public String getSelectCOMObjectsQueryString() {
+        return SELECT_COM_OBJECTS;
```

Comment:
Returning prepared statement query string is suspicious functionality and lets the user go against prepared statement model.